app.js

```diff
@@ -10,16 +10,32 @@ const upload = multer({
 const dotenv = require('dotenv');
 dotenv.config();
 
-
 app.set('view engine', 'ejs');
 app.use(express.static('static'));
 
+// Mongoose
+const mongo = require('mongodb')
+const mongoose = require('mongoose')
+
+const db = mongoose.connection
+
+// Connect mongoose with the database
+mongoose.connect(process.env.DB_URI, {
+  useNewUrlParser: true, 
+  useUnifiedTopology: true
+})
+
+db.on('connected', () => { 
+  console.log('Mongoose connected')
+})
 
 // Mongodb
 const MongoClient = require('mongodb').MongoClient;
 const uri = process.env.DB_URI;
 const client = new MongoClient(uri,{ useUnifiedTopology: true });
```

Comment:
If we were to use mongoose over MongoDB, we should remove anything that makes a connection through MongoDB. Don't worry about dependencies (in `package.json`), I'll make sure that works. For now just make sure variables and references to MongoDB are removed.